src/colorizer/UrlUtility.ts

```diff
@@ -0,0 +1,68 @@
+const URL_PARAM_TRACK = "track";
+const URL_PARAM_DATASET = "dataset";
+const URL_PARAM_FEATURE = "feature";
+const URL_PARAM_TIME = "t";
+
+export default class UrlUtility {
+  /**
+   * Updates the current URL path of the webpage. If any parameter value is falsy (null), it will not be
+   * included.
+   * @param dataset
+   * @param feature
+   * @param track
+   * @param time
+   */
+  public static updateURL(
+    dataset: string | null,
+    feature: string | null,
+    track: number | null,
+    time: number | null
+  ): void {
+    const params: string[] = [];
+    // Get parameters, ignoring null/empty values
+    if (dataset) {
+      params.push(`${URL_PARAM_DATASET}=${dataset}`);
+    }
+    if (feature) {
+      params.push(`${URL_PARAM_FEATURE}=${feature}`);
+    }
+    if (track) {
+      params.push(`${URL_PARAM_TRACK}=${track}`);
+    }
+    if (time) {
+      params.push(`${URL_PARAM_TIME}=${time}`);
+    }
```

Comment:
is it possible for `track` or `time` to be 0 and skip these ifs?

Reply:
good catch. definitely true for time at least.

Reply:
I think time = 0 isn't a huge deal because that's the default loaded frame. I want to say track values of 0 are also unused but I'll update this if check to allow them. Thanks!

Reply:
Nope I stand corrected, track=0 values are used!